docs/basics/tx-lifecycle.md

```diff
@@ -184,30 +185,30 @@ to during consensus. Under the hood, `DeliverTx` is almost identical to `CheckTx
 [`runTx`](../core/baseapp.md#runtx) function in deliver mode instead of check mode.
 Instead of using their `checkState`, full-nodes use `deliverState`:
 
-* **Decoding:** Since `DeliverTx` is an ABCI call, `Tx` is received in the encoded `[]byte` form.
-Nodes first unmarshal the transaction, then call `runTx` in `runTxModeDeliver`, which is very
-similar to `CheckTx` but also executes and writes state changes.
+- **Decoding:** Since `DeliverTx` is an ABCI call, `Tx` is received in the encoded `[]byte` form.
```

Comment:
I'd mention how the default tx encoder/decoder on the SDK work too

Reply:
I added a mention of TxConfig. I would rather go into more details in the Tx dedicated page.